### DISCUSSION TITLE
Emulating Windows 10/11 is very slow and demanding

### DISCUSSION THREAD

**Discussion by Lars-ellery02** [utmapp/UTM#3695] [UTM Help (macOS)] [answered]
UTM version: 3.0.4
MacOS: 12.2.1
Apple Silicon M1

I have not done any in depth testing, but when I try and emulate Windows 10/11 it’s slows down to almost stopping, and it immediately goes to full ram usage. 
Any hints?

**Comment by osy** [comment, marked as answer]
Yes this is expected. You are emulating a modern OS and likely on a single core. I would recommend using an older OS like Windows 7. Alternatively try virtualizing ARM Windows 11.
